app/controllers/campaigns_controller.rb

```diff
@@ -1,12 +1,26 @@
 # frozen_string_literal: true
+include Pagy::Backend
 
 class CampaignsController < ApplicationController
   before_action :set_campaign, only: %i[show update]
+  after_action { pagy_headers_merge(@pagy) if @pagy }
 
   # GET /campaigns
   def index
-    @campaigns = valid_campaigns.order(:end_date).all
-    json_response(campaigns_json)
+    # @NOTE(wilson) check to see if querying for inactive campaigns
+    # otherwise return active campaigns by default
+    inactive = params[:inactive]
+    if inactive == 'true'
```

Comment:
worried about boolean / string logic here -- should we try to convert then check it as a boolean to avoid potential issues? Not sure how we parse url query params

Reply:
I can double check that, but yeah it's probably better to convert to a boolean. I'll fix that before marking the PR up for review